src/python/CMakeLists.txt

```diff
@@ -8,23 +8,10 @@ MACRO(PYTHON_UNSET)
 ENDMACRO(PYTHON_UNSET)
 
 if (NOT SKBUILD)
-    if (${PYTHON_DESIRED} STREQUAL "2")
-        PYTHON_UNSET()
-        FIND_PACKAGE(PythonLibs 2 EXACT)
-        FIND_PACKAGE(PythonInterp 2 EXACT REQUIRED)
-    elseif (${PYTHON_DESIRED} STREQUAL "3")
-        PYTHON_UNSET()
-        SET(Python_ADDITIONAL_VERSIONS 3.0 CACHE INTERNAL "")
-        FIND_PACKAGE(PythonLibs 3 EXACT)
-        FIND_PACKAGE(PythonInterp 3 EXACT REQUIRED)
-    elseif (EXISTS ${PYTHON_DESIRED})
-        PYTHON_UNSET()
-        SET(PYTHON_EXECUTABLE ${PYTHON_DESIRED} CACHE INTERNAL "")
-        FIND_PACKAGE(PythonInterp REQUIRED)
-        FIND_PACKAGE(PythonLibs)
-    else ()
-        MESSAGE(FATAL_ERROR "Invalid PYTHON_DESIRED value: " ${PYTHON_DESIRED})
-    endif ()
+    PYTHON_UNSET()
+    SET(Python_ADDITIONAL_VERSIONS 3.0 CACHE INTERNAL "")
```

Comment:
FWIW do you know what these do and if they're still needed? Or did you just keep the original python 3 branch the way it was?

Reply:
I just kept it the same, not entirely sure why it was here originally.